### PR TITLE
WRP-2861: Add a menu linking to enactjs.com/sampler

### PR DESCRIPTION
**Pull request by sjro**
Enact-DCO-1.0-Signed-off-by: SJ RO (sj.ro@lge.com)

Currently. there is no menu on enactjs.com to view the sampler.
People who do not know the entire url (enactjs.com/sampler) cannot access it.
So add 'SAMPLER' menu.

### CODE REVIEW

File: src/components/SiteHeader/SiteHeader.js [hunk 1]
```diff
@@ -86,6 +86,13 @@ const SiteHeaderBase = kind({
 								>
 									Github
 								</Cell>
+								<Cell
+									component={OutboundLink}
+									shrink
+									href="https://enactjs.com/sampler"
+								>
+									Sampler
```
Comment (seunghoh) [inline src/components/SiteHeader/SiteHeader.js:94]: I prefer `UI Components` or `Components`.

**Review by seunghoh — COMMENTED** [review]
I agree with the new menu location but `Sampler` looks too our internal term

File: src/components/SiteHeader/SiteHeader.js (hunk 1 above)
Comment (sjro) [reply src/components/SiteHeader/SiteHeader.js]: Fixed it.

**Review by seunghoh — APPROVED** [review]
LGTM

**Pull request merged**